lib/view/icu_non_intensivist/general_care_view.dart

```diff
@@ -4,7 +4,7 @@ import 'package:flutter/widgets.dart';
 import '../../style.dart';
 
 class GeneralCareView extends StatelessWidget {
-  final title = 'General Care';
```

Comment:
duplicated change, there's a same string in Hard Data. Are we consider using a string resource file so we can avoid these duplicated strings?

Reply:
Totally agree. We should have a file to store all the static strings.

Reply:
Totally agree. We should have a file to store all the static strings.

Reply:
Yep, @workerbee22 is on the case!  We should be using a string resource file

Reply:
If @workerbee22 is on it, then I'll update this part later.